assignments/lambda-classes.js

```diff
@@ -1 +1,211 @@
 // CODE here for your Lambda Classes
+
+/* 
+
+Roster of Lambda personnel:
+Lambda personnel can be broken down into three different types of people:
+Instructors - extensions of Person, Students - extensions of Person, Project Managers - extensions of Instructors
+
+*/
+
+class Person {
+  constructor(attributes) {
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+    this.gender = attributes.gender;
+  }
+  speak() {
+    console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+  }
+}
+
+class Instructor extends Person {
+  constructor(attributes2) {
+    super(attributes2);
+    this.specialty = attributes2.specialty;
+    this.favLanguage = attributes2.favLanguage;
+    this.catchPhrase = attributes2.catchPhrase;
+  }
+  demo(subject) {
```

Comment:
Also, for future projects consider console.log to be a tool for checking variable values and error logs, and some other things, However, for functions and methods  - they have to return something and unlike in other languages the last line in a block does not return implicitly .. the only situation where a return occurs implicitly without the reserved return keyword is when you have a one line only arrow function.

Reply:
In fact, when apps move from development to staging and production, the console.logs have to be cleaned up and removed and will not be in the production code.  Just something, I wanted to mention since I too had run into this issue as couldn't for a while figure out some bugs that drove me nuts.